tests/stdlib/tmacros.nim

```diff
@@ -0,0 +1,9 @@
+import macros
+
+block: # hasArgOfName
+  macro m(u:untyped): untyped =
```

Comment:
also spacing here: `m(u: untyped): untyped =` (can be fixed in subsequent cleanup, or here)